modules/test/playwright/utils/getRandomKeyString.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {v4 as uuidv4} from 'uuid';
+
+export default function getRandomKeyString(): string {
```

Comment:
As we are using this util only once, it doesn't need to be a util. You can just add code inline.

Conversely, if we did use it multiple times in DSM, the correct place for it would be [local project utils dir](https://github.com/liferay/liferay-portal/tree/master/modules/test/playwright/tests/frontend-data-set-admin-web/utils), not global dir for all PW projects.

Reply:
I've remove it and added code inline